src/integration.rs

```diff
@@ -0,0 +1,396 @@
+// Copyright © 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Integration tests live in this file. We can't use the Rust "integration test" mode
+// as we don't have the expected source code structure.
```

Comment:
So what prevents our use of integration tests here? It seems like this file could just live in `tests/integration.rs`, and we could then avoid adding a Cargo feature.

We would still have to run `cargo xbuild` first, but that seems fine.

Reply:
I couldn't get it to work. It always tries to build the contents of the src with the normal toolchain.